.circleci/config.yml

```diff
@@ -41,18 +41,20 @@ jobs:
       - checkout
       - run: git submodule sync
       - run: git submodule update --init
+      - run: git fetch origin main  # needed for comparisson in pre-commit
+      - run: git branch --track main origin/main  # needed for comparisson in pre-commit
       - run: pip install --user tox
+      - run: tox -e pre-commit
+      - run: tox -e migrations
```

Comment:
Any reason we swapped these around?

Reply:
I saw the build times and migrations had a long running time, so in order to have a fail-fast for linting, I put it higher.

I think it should ultimately have its own pipeline.